repostats/repostats.py

```diff
@@ -1082,10 +1090,50 @@ def get_astroconda_membership(name="", data=""):
     return False
 
 
+def scrap_rtd_badge(repoorg, reponame):
```

Comment:
I'd rather keep the same naming convension throughout the library: repoorg--> org, reponame-->name
also, set the defaults to None and catch the exception

move "scrap" to "scrape"

Reply:
IMHO when arguments are not optional, they should be required positional ones. However, since you are the supreme ruler here, I'll make the change. :)

Reply:
this is true, just rename to be consistent at least